exp_configs/cluster/openai-gpt-3.5-turbo.json

```diff
@@ -0,0 +1,15 @@
+{
+    "use_slurm": true,
+    "ntasks": 1,
+    "cpus_per_task": 1,
+    "n_gpus": 1,
+    "gpu_type": "T4",
+    "device_id": "auto",
+    "mem": "16GB",
```

Comment:
Not sure if assigning GPUs is necessary here, but not really familiar with SLURM overall.

Reply:
This is only necessary for the evaluation part of the pipeline. A few of the metrics require a GPU, so it's specified here for ease of execution. The logic in the `run.py` doesn't apply this to the inference part for API models.